projects/data-structures-algorithms-sprint.md

```diff
@@ -6,14 +6,14 @@ This sprint will also help you practice study skills, summarizing, teamwork, and
 
 
 ### Topics 
-- [Data Structures](/data-structures/intro-to-data-structures.md) 
-- [Stacks](/data-structures/stack.md) 
-- [Queues](/data-structures/queues.md) 
-- [Deques](/data-structures/deque.md) 
-- [Linked Lists](/data-structures/linked-lists.md) 
-- [Hash Tables](/data-structures/hash-tables.md) 
-- [Trees](/data-structures/trees.md) 
-- [Searching](/algorithms/searching.md) and [Sorting](/master/algorithms/sorting.md) Algorithms 
+- [Data Structures](../data-structures/intro-to-data-structures.md) 
+- [Stacks](../data-structures/stack.md) 
+- [Queues](../data-structures/queues.md) 
+- [Deques](../data-structures/deque.md) 
+- [Linked Lists](../data-structures/linked-lists.md) 
+- [Hash Tables](../data-structures/hash-tables.md) 
+- [Trees](../data-structures/trees.md) 
+- [Searching](../algorithms/searching.md) and [Sorting](../algorithms/sorting.md) Algorithms 
```

Comment:
Please leave these unchanged.

Reply:
If i unchanged these links... those links are not working...is this okay?

Reply:
Oh I see what you mean. The correct fix, I believe, is to remove the leading `/` then. Thanks for catching that!

Reply:
If we keep leading slash also not working, so i left as it is like 
../data-structures/stack.md